src/IconButton/README.md

```diff
@@ -52,24 +53,81 @@ notes: ''
 ```jsx live
 () => {
   const icon = FontAwesome.faBars;
-  const variants = ["brand", "primary", "secondary", "success", "warning", "danger", "light", "dark"];
 
   return (
     <div className="d-flex">
-      {variants.map((variant) => {
-        return (
-          <div class={`bg-${variant} p-1`}>
-            <IconButton
-              key={variant}
-              icon={icon}
-              alt='Menu'
-              onClick={() => console.log("You clicked the menu button")}
-              variant={variant}
-              invertColors
-            />
-          </div>
-        );
-      })}
+      <div class="p-1 bg-brand">
+        <IconButton
+          icon={icon}
+          alt='Menu'
```

Comment:
nit: Some props in these examples use double quotes for their values and others use single quotes. We should align on using one or the other, but not both.

Reply:
The standard I tend to hold to is double-quotes for props, single quotes for strings.

Reply:
Same here :)